CHANGELOG.md

```diff
@@ -5,29 +5,28 @@ CHANGELOG
 -----
 
 **ENHANCEMENTS**
-- Add possibility to specify multiple instance types for the same compute resource.
+- Add possibility to specify multiple EC2 instance types for the same compute resource.
 - Add support for adding and removing shared storages at cluster update by updating `SharedStorage` configuration.
-- Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage
-  to support storage retention on deletion.
-- Add support for Slurm Accounting.
-- Add support for on-demand capacity reservations.
-- Add support for specifying the supported IMDS version in cluster and build image configurations via the `Imds/ImdsSettings` property.
+- Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage to support storage retention on deletion.
+- Add new configuration to enable accounting functionality in Slurm.
+- Add support for On-Demand Capacity Reservations and Capacity Reservations Resource Groups.
+- Add new configuration parameter in `Imds/ImdsSettings` to specify the IMDS version to support in a cluster or build image infrastructure. 
+- Add support for `Networking/PlacementGroup` in the `SlurmQueues/ComputeResources` section.
 - Add support for instances with multiple network interfaces that allows only one ENI per device.
 - Improve validation of networking for external EFS file systems by checking the CIDR block in the attached security group.
-- Add validator for instances being launched in a cluster placement group when the instance type does not support it.
+- Add validator to check if configured instance types support placement groups.
```

Comment:
Minor: we used "Capacity Reservations Resource Group" (upper case first letter), so we should do the same for placement groups?